test-app/shared/package-lock.json

```diff
@@ -1,1508 +1,8 @@
 {
   "name": "@general-vms/shared",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This is still an issue.

Reply:
uh oh, probably because I cherry-picked only my original commit. Will fix